pwa/Dockerfile

```diff
@@ -20,10 +20,10 @@ RUN corepack enable && \
 # Deps stage, preserve dependencies in cache as long as the lockfile isn't changed
 FROM builder_base AS deps
 
-COPY --link pnpm-lock.yaml ./
```

Comment:
why doens't link work?

Reply:
The was a problem when loading the old con data in the pwa, I test to solve this issue by remove the --link on a colleague advise. The issue was somewere else, I can restore the --link I guess

Reply:
https://github.com/api-platform/website/pull/375/commits/363a004e9184556be6dbdcbb1f0c7808bb47d5d7